src/solvers/flattening/boolbv_onehot.cpp

```diff
@@ -11,6 +11,8 @@ Author: Daniel Kroening, kroening@kroening.com
 
 literalt boolbvt::convert_onehot(const unary_exprt &expr)
 {
+  PRECONDITION(expr.id() == ID_onehot || expr.id() == ID_onehot0);
```

Comment:
Are those expressions used anywhere? The IDs are not assigned anywhere.

Reply:
They are used in ebmc.

Reply:
Raised an issue for those as well: #3040